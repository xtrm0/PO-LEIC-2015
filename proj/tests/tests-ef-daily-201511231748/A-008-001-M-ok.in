5
JoÃ£o Pereira
j@com
4
0
