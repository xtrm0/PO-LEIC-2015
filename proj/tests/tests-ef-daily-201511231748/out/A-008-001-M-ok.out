Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Nome do autor: Email do autor: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Título: 
Autor: João Pereira/j@com
Secções de topo: 0
Dimensão do documento (bytes): 0
Identificadores únicos: 0
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: 